### DISCUSSION TITLE
install on ubuntu 18.04

### DISCUSSION THREAD

**Discussion by rinoshjames** [zephir-lang/zephir#2291] [Q&A] [answered]
how to install zephir on ubuntu 18.04

**Comment by Jeckerson** [comment]
Zephir
a. Download phar file from releases - https://github.com/zephir-lang/zephir/releases
b. composer require --dev `phalcon/zephir`

Zephir parser
```
pecl install zephir_parser
```

**Reply by rinoshjames** [reply]
download zephir.phar

what is the next command?

**Comment by Jeckerson** [comment, marked as answer]
Here is example of Zephir and Zephir parser installation inside PHP docker container (`php:8.0-fpm`).

https://user-images.githubusercontent.com/3289702/134533010-16dbfd8c-3734-4ff4-80b4-79b7ab5e0e10.mov

**Reply by rinoshjames** [reply]
thank you very much...

**Reply by rinoshjames** [reply]
how to build using php zephir.phar
